contribute/ide.setup.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: "Beam Contribution Guide"
+permalink: /contribute/ide-setup/
+---
+
+# Apache Beam IDE setup
+
+* TOC
+{:toc}
+
+Explains how to setup your IDE and import the beam code so you can work on it. It assumes that you have already checked
+out the code.
+
+## Intellij
+
```

Comment:
Maybe just a quick comment about the code style would be helpful there. I can do that when merging the PR.